packages/visual-editor/src/hooks/usePlaceholderStyle.ts

```diff
@@ -136,15 +136,21 @@ export const usePlaceholderStyle = () => {
         clientX = offsetLeft - translateValue;
       }
 
+      const draggingNewComponent = !!draggedItem.source.droppableId.startsWith('component-list');
+      const shouldShowBackgroundColor = draggingNewComponent;
```

Comment:
small nit pick: `draggingNewComponent` var could be renamed to avoid the alias definition
```suggestion
      const shouldShowBackgroundColor = !!draggedItem.source.droppableId.startsWith('component-list');
```